N3.js/src/main/scala/org/w3/banana/n3js/N3.scala

```diff
@@ -0,0 +1,194 @@
+package org.w3.banana
+package n3js
+
+import scala.scalajs.js
+
+object N3 extends js.Object {
+
```

Comment:
I do not understand your remark.

The only relevant information is the *Top-level JavaScript objects* section in http://www.scala-js.org/doc/calling-javascript.html